### PR TITLE
Sdb msvc fixes (regression)

### PR DESCRIPTION
**Pull request by xarkes**

### CODE REVIEW

File: shlr/sdb/src/cdb.c [hunk 1]
```diff
@@ -101,7 +101,7 @@ bool cdb_read(struct cdb *c, char *buf, ut32 len, ut32 pos) {
 		return false;
 	}
 	while (len > 0) {
-		ssize_t r = read (c->fd, buf, len);
+		ut32 r = read (c->fd, buf, len);
```
Comment (radare) [inline shlr/sdb/src/cdb.c]: Thats wring. It returns signed. And ssize_t should be ok. Whats the signature on windows?

File: shlr/sdb/src/sdb.h [hunk 2]
```diff
@@ -92,7 +94,6 @@ typedef struct sdb_t {
 	SdbList *hooks;
 	SdbKv tmpkv;
 	ut32 depth;
-	bool timestamped;
```
Comment (radare) [inline shlr/sdb/src/sdb.h]: Wtf dont regress

File: shlr/sdb/src/json/js0n.c [hunk 3]
```diff
@@ -5,6 +5,12 @@
 
 #include "rangstr.h"
 
+#ifdef _MSC_VER
+#pragma message ("TODO: json not implemented for this platform")
+int js0n(const ut8 *js, RangstrType len, RangstrType *out) {
```
Comment (radare) [inline shlr/sdb/src/json/js0n.c]: Seems like a bad idea

**Comment by Maijin** [comment]
doesn't build https://travis-ci.org/radare/radare2/builds/236716026?utm_source=github_status&utm_medium=notification

**Comment by xarkes** [comment]
I know

**Comment by xarkes** [comment]
I close it it's related to sdb not to radare

**Pull request closed without merging**